packages/arb-token-bridge-ui/src/util/TokenUtils.ts

```diff
@@ -235,3 +238,49 @@ export async function l1TokenIsDisabled({
   const erc20Bridger = await Erc20Bridger.fromProvider(l2Provider)
   return erc20Bridger.l1TokenIsDisabled(erc20L1Address, l1Provider)
 }
+
+type SanitizeTokenOptions = {
+  erc20L1Address?: string | null // token address on L1
+  chain: Chain // chain for which we want to retrieve the token name / symbol
+}
+
+// get the exact token symbol for a particular chain
+export function sanitizeTokenSymbol(
+  tokenSymbol: string,
+  options?: SanitizeTokenOptions
+) {
+  if (typeof options === 'undefined') {
+    return tokenSymbol.toUpperCase()
+  }
+
+  const isArbitrumOne = isNetwork(options.chain.id).isArbitrumOne
+
+  // only special case for USDC is Arbitrum One
+  if (options.erc20L1Address === CommonAddress.Mainnet.USDC && isArbitrumOne) {
```

Comment:
```suggestion
  if (options.erc20L1Address.toLowerCase() === CommonAddress.Mainnet.USDC.toLowerCase() && isArbitrumOne) {
```

Reply:
We should lowercase the common addresses in that file

Reply:
`CommonAddress.Mainnet.USDC` is already lowercased.